src/qililab/instruments/quantum_machines/quantum_machines_cluster.py

```diff
@@ -21,7 +21,7 @@
 from qm import DictQuaConfig, QuantumMachine, QuantumMachinesManager, SimulationConfig
 from qm.jobs.running_qm_job import RunningQmJob
 from qm.octave import QmOctaveConfig
-from qm.qua import Program
+from qm.program import Program
```

Comment:
Was this change documented? I cannot find any references into it. How did you find out that this needed to change

Reply:
I think its just an improvement, since both get you to the same final class, but the former one, passed at least two extra files, with a lot of imports.. 

(which makes things slower and more circular import error prone I guess 🤔 )

Reply:
I guess he saw with VSCode checkers, that the import wasn;t being successful..

And then if you go back to the old version of `qm-qua`, that class is the one that was being imported before (after a couple of file jumps 🙃)